### PR TITLE
Add Devise and ActiveAdmin gems

### PR DESCRIPTION
**Pull request by AgostinaDev**
Devise and Active Admin were added to create admin section
Trello card: https://trello.com/c/Vz2ROL4y

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+ActiveAdmin.register_page 'Dashboard' do
```
Comment (federicoaldunate) [inline app/admin/dashboard.rb]: I don't think we need this on this PR, what do you think?

File: app/models/admin_user.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#
+# Indexes
+#
+#  index_admin_users_on_email                 (email) UNIQUE
+#  index_admin_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
+class AdminUser < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable,
```
Comment (federicoaldunate) [inline app/models/admin_user.rb:22]: Why do you have all this code repeated?

File: db/migrate/20210329180842_create_active_admin_comments.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class CreateActiveAdminComments < ActiveRecord::Migration[6.1]
```
Comment (federicoaldunate) [inline db/migrate/20210329180842_create_active_admin_comments.rb]: Do we need this migration and/or this model?

File: db/seeds.rb [hunk 4]
```diff
@@ -6,7 +6,11 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-topic_descriptions = %w[Music\ Sports\ Videogames\ Gardening\ Travelling\ Space]
-topic_descriptions do |description|
+topic_descriptions = %w[Music Sports Videogames Gardening Travelling Space]
+topic_descriptions.each do |description|
   topic = Topic.find_or_create_by!(description: description)
```
Comment (federicoaldunate) [inline db/seeds.rb]: do you need to assign the result to that `topic` variable?
what about `Topic.create(description: description)` without `!` so errors don't raise

File: spec/models/admin_user_spec.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+#  reset_password_token   :string
+#
+# Indexes
+#
+#  index_admin_users_on_email                 (email) UNIQUE
+#  index_admin_users_on_reset_password_token  (reset_password_token) UNIQUE
+#
+require 'rails_helper'
+
+RSpec.describe AdminUser, type: :model do
```
Comment (federicoaldunate) [inline spec/models/admin_user_spec.rb]: remember to not add files that are not being used yet

File: app/assets/stylesheets/active_admin.scss [hunk 6]
```diff
@@ -0,0 +1,17 @@
+// Sass variable overrides must be declared before loading up Active Admin's styles.
```
Comment (TimoPeraza) [inline app/assets/stylesheets/active_admin.scss:1]: don't you have webpacker in your rails project?

**Review by federicoaldunate — APPROVED** [review]
Nice work!

**Review by TimoPeraza — APPROVED** [review]
🏅

**Pull request merged**